src/clj/tasks/load_generator.clj

```diff
@@ -0,0 +1,141 @@
+            [clj-uuid :as uuid]
+            [org.httpkit.client :as http]
+            [web.core :refer [-main]]
+            [web.ws :as game-ws-handler]
+            [monger.collection :as mc]
+            [clojure.tools.cli :refer [parse-opts]]
+            [web.db :refer [db]]
+            [web.lobby :refer [all-games]])
+  (:import java.net.URLEncoder))
+
+;; This print guarantees a coherent print (i.e. parallel prints will not be interleaved)
+(defn safe-println [& more]
+  (.write *out* (str (join " " more) "\n")))
+
+(defn login
+  [username password]
+  (let [options {:form-params {:username username
+                          :password password}}
+    post-res @(http/post "http://localhost:1042/login" options)]
+    (if (or (post-res :error) (= 401 (post-res :status)))
+      (println "Failed, exception is " (or (post-res :error) (post-res :status)))
+        (let [get-res @(http/get "http://localhost:1042" {:as :text :headers {"Cookie" (str (:set-cookie (post-res :headers)))}})]
+          (if (or (get-res :error) (= 401 (get-res :status)))
+            (println "Failed, exception is " (or (get-res :error) (get-res :status)))
+              {"Origin" "http://localhost:1042" "Cookie" (str (:set-cookie (post-res :headers)) ";" (:set-cookie (get-res :headers))) "X-CSRF-Token" (second (re-find #"data-csrf-token=\"(.*?)\"" (str (get-res :body))))})))))
+
+(defn create-game
+  [maxUsers]
+  (let [client (ws/client)
+        corp-client-id (uuid/to-string (uuid/v1))
+        runner-client-id (uuid/to-string (uuid/v1))
+        corp-login (login "sample2" "password")
+        runner-login (login "sample3" "password")
+        corp-decks (mc/find-maps db "decks" {:username "sample2"})
+        runner-decks (mc/find-maps db "decks" {:username "sample3"})]
+    (.setMaxTextMessageSize (.getPolicy client) (* 1024 1024))
+    (.start client)
+
+    (safe-println "Login Corp")
+      (ws/connect (str "ws://localhost:1042/ws?client-id=" corp-client-id "&csrf-token=" (URLEncoder/encode (corp-login "X-CSRF-Token")))
+                :client client
+                :extensions ["permessage-deflate"]
+                :on-error #(safe-println "corp error" %)
+                :on-connect (fn [n] (safe-println "Corp Connected"))
+                :on-close (fn [x y] (safe-println "Corp Disconnected"))
+                :headers corp-login)
+    (safe-println "Login Runner")
+      (ws/connect (str "ws://localhost:1042/ws?client-id=" runner-client-id "&csrf-token=" (URLEncoder/encode (runner-login "X-CSRF-Token")))
+                :client client
+                :on-error #(safe-println "runner error" %)
+                :on-connect (fn [n] (safe-println "Runner Connected"))
+                :on-close (fn [x y] (safe-println "Runner Disconnected"))
+                :headers runner-login)
+    (safe-println "Create lobby")
+
+    ;; Corp create lobby
+    (game-ws-handler/-msg-handler {:id :lobby/create :ring-req {:user {:username "sample2"}}
```

Comment:
spacing in these sections. maybe put the `:ring-req` on the next line for clarity.